src/pages/Home.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div>
+    <carousel-list
+      v-if="productList.length"
```

Comment:
Should probably reference relevant data? 🤔 
```suggestion
      v-if="carouselListData.length"
```